tests/druid_tests.py

```diff
@@ -90,12 +90,12 @@ def __reduce__(self):
     {
         "version": "v1",
         "timestamp": "2012-01-01T00:00:00.000Z",
-        "event": {"dim1": "Canada", "dim2": "boy", "metric1": 12345678},
+        "event": {"dim1": "Canada", "dim2": "boy", "count": 12345678},
```

Comment:
This didn't represent the correct result set when using the `count` metric. Previously this wasn't a problem because the Table viz would throw out columns not in the data frame and the relevant tests don't check the associated metric column.